rb_ws/src/buggy/buggy/serial/ros_to_bnyahaj.py

```diff
@@ -143,14 +143,14 @@ def reader_thread(self):
         self.get_logger().info("Starting reading odom from teensy!")
         while rclpy.ok():
             packet = self.comms.read_packet()
+            self.get_logger().debug("packet" + str(packet))
 
             if isinstance(packet, Odometry):
-                self.get_logger().debug("packet" + str(packet))
                 # Publish to odom topic x and y coord
                 odom = ROSOdom()
                 # convert to long lat
                 try:
-                    lat, long = World.utm_to_gps(packet.y, packet.x)
+                    lat, long = utm.to_latlon(packet.x, packet.y, 17, "T")
```

Comment:
Did you by chance swap the order (this is actually super tricky and make sure we match it up)

Reply:
Restored to original ordering and added a TODO comment